From: Johannes Thumshirn <jthumshirn@suse.de>
Subject: kabi protect enum mem_type
Date: Fri Oct  5 09:34:08 CEST 2018
Patch-mainline: Never, kabi fixup
References: bsc#1099125

Kabi fixup for patch patches.drivers/EDAC-Add-new-memory-type-for-non-volatile-DIMMs.patch.

Signed-off-by: Johannes Thumshirn <jthumshirn@suse.de>
---
 include/linux/edac.h |    2 ++
 1 file changed, 2 insertions(+)

--- a/include/linux/edac.h
+++ b/include/linux/edac.h
@@ -210,7 +210,9 @@ enum mem_type {
 	MEM_DDR4,
 	MEM_RDDR4,
 	MEM_LRDDR4,
+#ifndef __GENKSYMS__
 	MEM_NVDIMM,
+#endif
 };
 
 #define MEM_FLAG_EMPTY		BIT(MEM_EMPTY)
